sql-plugin/src/main/scala/com/nvidia/spark/rapids/SpillableKudoTable.scala

```diff
@@ -0,0 +1,60 @@
+package com.nvidia.spark.rapids
+
+import ai.rapids.cudf.HostMemoryBuffer
+import com.nvidia.spark.rapids.jni.kudo.KudoTable
+import com.nvidia.spark.rapids.jni.kudo.KudoTableHeader
+
+/*
```

Comment:
this copyright needs to be at the top of the file.

Reply:
interesting,,, I guess the imports are auto completed and the maven plugin didn't warn on this... fixed